concepts/choosing_a_dataset.md

```diff
@@ -68,6 +68,17 @@ This data is available in the **telemetry_new_profile_parquet** dataset.
 
 {% include "/datasets/batch_view/new_profile/intro.md" %}
 
+# Update Derived Dataset
+
+The [update ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/update-ping.html)
```

Comment:
This summary of the update ping is super useful. Do you mind moving lines 73-76 into intro.md? That way this summary will be included in the reference documentation as well.